File: 282A - Bit++.py
def bit(inp):
  value = 0
  for i in range(inp):
    op = input()
    if '++' in op or op == 'x++' or op == '++x':
        value += 1
    elif '--' in op or op == 'x--' or op == '--x':
        value -= 1
  print(value)
bit(int(input()))

# Alternative -> Better Solution

def bit(inp):
    value = 0
    for i in range(inp):
        op = input()
        if '++' in op:
            value += 1
        elif '--' in op:
            value -= 1
    print(value)

bit(int(input()))

File: 282A - Bit++.js
function bit(inp) {
  var value = 0;
  for (i = 0; i < inp; i++) {
    var op = readline();
    if (op.includes("++") || op === "++x" || op === "x++") {
      value += 1;
    } else if (op.includes("--") || op === "--x" || op === "x--") {
      value -= 1;
    }
  }

  print(value);
}

bit(parseInt(readline()));

// Alternative -> Efficient Solution

function bit(inp) {
  var value = 0;
  for (i = 0; i < inp; i++) {
    var op = readline();
    if (op.includes("++")) {
      value += 1;
    } else if (op.includes("--")) {
      value -= 1;
    }
  }

  print(value);
}

bit(parseInt(readline()));
